src/components/navigation-bar/navigation-bar.jsx

```diff
@@ -36,7 +41,7 @@ export const NavigationBar = ({ user, onLoggedOut }) => {
                 <Nav.Link className='link-header-light' as={Link} to='/profile'>
                   Profile
                 </Nav.Link>
-                <Nav.Link onClick={onLoggedOut} className='ms-lg-auto'>
+                <Nav.Link onClick={() => dispatch(setUser(null),setToken(null),localStorage.clear(),window.location.reload())} className='ms-lg-auto'>
```

Comment:
⛏️If you can always extract this to a single function to simplify the complexity of this line.

Reply:
done! thanks